Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -321,12 +338,20 @@ open class Kommunicate: NSObject, Localizable {
                 completionHandler(false)
                 return
             }
-            self.openChatWith(
-                groupId: key,
-                from: viewController,
-                prefilledMessage: prefilledMessage,
-                showListOnBack: showListOnBack
-            ) { result in
+            guard let rootView = rootView
```

Comment:
For create and embed we've separate functions, but for `showConversation` we are using the old one. Can we create a function for this as well and keep all container related logic separate?

Reply:
done